zeekctl/af_packet.py

```diff
@@ -31,12 +42,9 @@ def zeekctl_config(self):
 
 			params = ""
 
-			if nn.af_packet_fanout_id:
-				params += "\n  redef AF_Packet::fanout_id = %s;" % nn.af_packet_fanout_id
-			if nn.af_packet_fanout_mode:
-				params += "\n  redef AF_Packet::fanout_mode = %s;" % nn.af_packet_fanout_mode
-			if nn.af_packet_buffer_size:
-				params += "\n  redef AF_Packet::buffer_size = %s;" % nn.af_packet_buffer_size
+			for p, v in vars(nn).items():
```

Comment:
Uhh, vars magic :-)

IMO this would be a bit more usual:
```
for key in self.nodeKeys():
   prefixed_key = f"{self.prefix()}_{key}"
   v = getattr(nn, prefixed_key, None)
   if v and v.strip():
       params += f"\n    redef AF_Packet::{key} = {v.strip()};"
```

not tested..

Reply:
🤔 Is the use of `vars()` considered bad practice? Using `self.prefix()` makes sense...